client/components/tabButton/tabButton.js

```diff
@@ -21,8 +21,8 @@ const A = styled.a`
   }
 `;
 
-const TabButton = ({ title, handleClick }) => (
-    <A onClick={handleClick}>{title}</A>
+const TabButton = ({ title, handleClick, featured }) => (
```

Comment:
Not important now, but might be interesting to consider how to eliminate these inline styles and use props in the styled component instead.

Reply:
yeah, I'm sure there's some way to do it with props